KOService.Application/Handlers/Client/SendRepairCreatedMailHandler.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using MediatR.Pipeline;
+using MediatR;
+using KOService.Application.Services;
+using KOService.Domain.DbContexts;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using KOService.Application.Commands.Repair;
+
+namespace KOService.Application.Handlers
+{
+    class SendRepairCreatedMailHandler : IRequestPostProcessor<CreateRepairCommand, Unit>
+    {
```

Comment:
ale tylko dwa dodałas hmm, a reszta ? czy ustatlismy ze tylko dal tych wysyłamy

Reply:
Znaczy jest po dodaniu, anulowaniu i zakończeniu i jak Tomek zmerguje to chciałam jeszcze dodać jeden, że gotowa wycena, coś jeszcze powinno być?

Reply:
aa ok :)